src/test/resources/feature/2-mittente/Sequence/invioNotificaConSequence_fail_AR.feature

```diff
@@ -24,7 +24,7 @@ Feature: invio notifica con sequence
     Then Creo in background una notifica per destinatario tramite API REST
     And Si seleziona la notifica mittente
     And Si attende completamento notifica "Depositata"
-    And Si attende completamento notifica
+    And Si attende la visualizzazione corretta del dettaglio della notifica
```

Comment:
what is this step for?

Reply:
questo step per attendere il caricamento del dettaglio della notifica

Reply:
non si attende giá nello step precedente

Reply:
lo step precedente non clicca sul dettaglio

Reply:
Modify